ci/lava/tests/test-bsp.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Script to run systemd tests on the DUT."""
+
+import pytest
+import re
+import subprocess
+
+
+class TestBsp:
+    """Class to encapsulate the testing of systemd on a DUT."""
+
+    dut_address = ""
+
+    def test_setup_dut_addr(self, dut_addr):
+        """Store the device address."""
+        TestBsp.dut_address = dut_addr
+
+        assert dut_addr != ""
+
+    def test_badblocks(self, execute_helper):
+        """Perform the test on the DUT via the mbl-cli."""
+        err, stdout, stderr = execute_helper.send_mbl_cli_command(
+            [
+                "shell",
+                'sh -l -c "badblocks -v '
+                "$(mount | grep \"on \/ type\" | cut -d' ' -f 1)"
```

Comment:
This line fails the sanity check. I think it's enough to do so (please test it):
```suggestion
                r"$(mount | grep \"on \/ type\" | cut -d' ' -f 1)"
```

Reply:
changed to use /sbin/blkid instead.